core/types/block_test.go

```diff
@@ -113,8 +113,8 @@ func TestEIP1559BlockEncoding(t *testing.T) {
 	check("Time", block.Time(), uint64(1426516743))
 	check("Size", block.Size(), uint64(len(blockEnc)))
 	check("BaseFee", block.BaseFee(), new(big.Int).SetUint64(1_000_000_000))
-	check("ExtDataGasUsed", block.ExtDataGasUsed(), (*big.Int)(nil))
-	check("BlockGasCost", block.BlockGasCost(), (*big.Int)(nil))
+	check("ExtDataGasUsed", BlockExtDataGasUsed(&block), (*big.Int)(nil))
```

Comment:
(optional) Having to always address it with `&block` suggests that it should be declared as a pointer in the first place. This would be in keeping with the values returned by constructors as well as all method receivers.

Reply:
This is geth code, so not changing this 😉 But otherwise good point